devicetypes/smartthings/ecolink-wireless-siren.src/ecolink-wireless-siren.groovy

```diff
@@ -144,6 +147,14 @@ def refresh() {
 	return delayBetween(cmds, 200)
 }
 
+def refreshChildren() {
```

Comment:
use this method to remove the duplicated code in `refresh()`

Reply:
done